README.md

```diff
@@ -15,3 +13,309 @@ To begin, clone this repository. `cd` into your chosen folder, and run the follo
 JavaScript in this tool is linted with [ESLint](http://eslint.org/) according to our code [syntax and style standards](https://github.com/mobify/mobify-code-style) here at Mobify.
 
 Linting may be run with the `grunt lint` command. Code is also linted automatically on [CircleCI](https://circleci.com/).
+
+## Assertions
+
+#### elementsCount(selector, expected, message, callback)
+
+The `elementsCount` assertion checks if the given selector is present the number of times that is expected for that selector to appear.
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
```

Comment:
@marlowpayne TIL thx, @ellenmobify if this is not a ton of work, I would actually like to implement this in this PR so that it is one less thing that we need to address later.